docs/content/modeling/migrating/migrating-schema-1-1.mdx

```diff
@@ -0,0 +1,494 @@
+
+<RelationshipTuplesViewer
+  relationshipTuples={[
+    {
+      user: 'document:pricing',
+      relation: 'member',
+      object: 'group:sales',
+      _description: 'The "pricing" document is a member of the "sales" group',
+    },
+  ]}
+/>
+
+To be able to better validate tuples and make the model more readable, version 1.1 requires you to specify types for all the relations that were previously assignable (e.g. relations defined `as self` in any way), and it removes the `as self` keyword.
+
+The model above needs to be rewritten as:
+
+<AuthzModelSnippetViewer
+  configuration={{
+    mode: {
+      schema_version: '1.1',
+    },
+    type_definitions: [
+      { type: 'user', relations: {} },
+      {
+        type: 'group',
+        relations: { member: { this: {} } },
+        metadata: {
+          relations: {
+            member: { directly_related_user_types: [{ type: 'user' }] },
+          },
+        },
+      },
+      {
+        type: 'folder',
+        relations: {
+          parent: { this: {} },
+          viewer: {
+            union: {
+              child: [
+                { this: {} },
+                {
+                  tupleToUserset: {
+                    tupleset: { object: '', relation: 'parent' },
+                    computedUserset: { object: '', relation: 'viewer' },
+                  },
+                },
+              ],
+            },
+          },
+        },
+        metadata: {
+          relations: {
+            parent: { directly_related_user_types: [{ type: 'folder' }] },
+            viewer: { directly_related_user_types: [{ type: 'user' }] },
+          },
+        },
+      },
+      {
+        type: 'document',
+        relations: {
+          parent: { this: {} },
+          viewer: { this: {} },
+          can_read: {
+            union: {
+              child: [
+                { computedUserset: { object: '', relation: 'viewer' } },
+                {
+                  tupleToUserset: {
+                    tupleset: { object: '', relation: 'parent' },
+                    computedUserset: { object: '', relation: 'viewer' },
+                  },
+                },
+              ],
+            },
+          },
+        },
+        metadata: {
+          relations: {
+            parent: { directly_related_user_types: [{ type: 'folder' }] },
+            viewer: { directly_related_user_types: [{ type: 'user' }] },
+            can_read: { directly_related_user_types: [] },
+          },
+        },
+      },
+    ],
+    schema_version: '1.1',
+  }}
+/>
+
+After making these changes, <ProductName format={ProductNameFormat.ShortForm}/> will start validating the tuples more strictly, for example, you won’t be able to assign a `document` as a member of a `group`. If your application is writing invalid tuples, you’ll start getting errors when invoking the `Write` API.
```

Comment:
Do you want to link to the API reference for the `Write` API so it's a link to the API? MIght be nice 🤷 ?